test/spec/controllers/foyer/ressources/yearMoins2.js

```diff
@@ -27,22 +26,6 @@ describe('Controller: FoyerRessourceYearMoins2Ctrl', function() {
             expect(scope.individuRefs[0].individu).toBe(demandeur);
             expect(scope.individuRefs[1].individu).toBe(conjoint);
         });
-
-        it('should init all amounts to zero', function() {
```

Comment:
Can't we change the test to something else? We remove coverage but don't add anything back.